sklearn/inspection/partial_dependence.py

```diff
@@ -433,14 +439,17 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
         - 'brute' is supported for any estimator, but is more
           computationally intensive.
 
-        - If 'auto', then 'recursion' will be used for
-          ``BaseGradientBoosting`` estimators with ``init=None``, and
-          'brute' for all other.
-
-        Unlike the 'brute' method, 'recursion' does not account for the
-        ``init`` predictor of the boosting process. In practice this still
-        produces the same plots, up to a constant offset in the target
-        response.
+        - 'auto':
+
+          - 'recursion' is used for
+            :class:`GradientBoostingClassifier<sklearn.ensemble.GradientBoostingClassifier>`
+            and
+            :class:`GradientBoostingRegressor<sklearn.ensemble.GradientBoostingRegressor>`
+            if ``init=None``, and for
+            :class:`HistGradientBoostingClassifier<sklearn.ensemble.HistGradientBoostingClassifier>`
+            and
+            :class:`HistGradientBoostingRegressor<sklearn.ensemble.HistGradientBoostingRegressor>`.
```

Comment:
Maybe we could make this less redundant and say that "auto" means "recursion" for estimators that are supported and "brute" otherwise.